portfolio.html

```diff
@@ -7,8 +7,8 @@
     <header>
       <div class="login">
         <ul>
-          <li><a href="index.html">Home</a></li>
           <li><a href="signup.html">login</a></li>
+          <li><a href="edit portfolio.html">edit Portfolio</a></li>
         </ul>
       </div>
     </header>
```

Comment:
move this out from here and into your `<body>` tag

Reply:
fixed!